packages/about/lib/update-manager.js

```diff
@@ -1,5 +1,10 @@
 const { Emitter, CompositeDisposable } = require('atom');
 
+const REPO_OWNER = process.env.REPO_OWNER || 'atom';
+const MAIN_REPO = process.env.MAIN_REPO || 'atom';
+const NIGHTLY_RELEASE_REPO =
+  process.env.NIGHTLY_RELEASE_REPO || 'atom-nightly-releases';
```

Comment:
This code is a runtime code (runs inside about package), and we can't rely on the environment variables for this. We should probably load the values from a separate file or hardcode the URLs